otel-extension/src/main/java/io/scalac/mesmer/otelextension/LoggingAgentExtension.java

```diff
@@ -0,0 +1,65 @@
+package io.scalac.mesmer.otelextension;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.tooling.AgentExtension;
+import java.io.File;
+import java.io.IOException;
+import net.bytebuddy.agent.builder.AgentBuilder;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.dynamic.DynamicType;
+import net.bytebuddy.utility.JavaModule;
+
+@AutoService(AgentExtension.class)
+public class LoggingAgentExtension implements AgentExtension {
```

Comment:
Please remove it. It seems not to be the part of the issue solution. 

If we want to have util like this: 
 - it should be managed independently from the library instrumentations. For example a separate otel agent extension that has to be deliberately turned on
 - it has to be parametrized so that there are no "piotrjosiak" file paths ;). What else should be parametrized here?
 - it needs documentation, such as running and configuration instructions. We will have some junior devs pretty soon - for sure this piece will be enigmatic 

So I suggest to remove it now and have it incorporated in another issue where we can focus on this feature and ✔️ all the bullets above